packages/flutter/lib/src/services/text_input.dart

```diff
@@ -824,7 +824,12 @@ abstract class TextInputClient {
   /// Requests that this client perform the given action.
   void performAction(TextInputAction action);
 
-  /// Requests that this client perform the private command.
+  /// Request from the input method that this client perform the given command.
```

Comment:
According to https://developer.android.com/reference/android/view/inputmethod/InputMethodSession#appPrivateCommand(java.lang.String,%20android.os.Bundle), this sends a command to the IME?

Reply:
I think performPrivateCommand is going from the input method to the app: https://developer.android.com/reference/android/view/inputmethod/InputConnection#performPrivateCommand(java.lang.String,%20android.os.Bundle)

And appPrivateCommand goes in the opposite direction.

Here's the original framework PR https://github.com/flutter/flutter/pull and engine PR https://github.com/flutter/engine/pull/20188.  Let me know if I could make this more clear in the docs.

Reply:
Ah I see. Then the description of `sendAppPrivateCommand` in the see also section is actually for `performPrivateCommand`?

Reply:
Yeah good catch.  I'll add a link to the docs for performPrivateCommand, too.